examples/sentence_transformers/simple.py

```diff
@@ -0,0 +1,22 @@
+import mlflow
+import mlflow.sentence_transformers
+
+from sentence_transformers import SentenceTransformer
+
+model = SentenceTransformer("all-MiniLM-L6-v2")
+
+# Log the model using mlflow
+with mlflow.start_run():
+    logged_model = mlflow.sentence_transformers.log_model(model, "sbert_model")
```

Comment:
Can we update this example to show:
1. logging with a declared signature
2. logging an input example
3. declaring the arg names for tutorial purposes (it helps for new users to the flavor)